docs/getting-started/config-options.md

```diff
@@ -174,8 +174,8 @@
 | PHOTOPRISM_FFMPEG_ENCODER        | --ffmpeg-encoder        | libx264                        | FFmpeg AVC encoder `NAME`                                             |
 | PHOTOPRISM_FFMPEG_SIZE           | --ffmpeg-size           |                           3840 | maximum video size in `PIXELS` (720-7680)                             |
 | PHOTOPRISM_FFMPEG_BITRATE        | --ffmpeg-bitrate        |                             50 | maximum video `BITRATE` in Mbit/s                                     |
-| PHOTOPRISM_FFMPEG_MAP_VIDEO      | --ffmpeg-map-video      | 0:v:0                          | video `STREAMS` that should be transcoded                             |
-| PHOTOPRISM_FFMPEG_MAP_AUDIO      | --ffmpeg-map-audio      | 0:a:0?                         | audio `STREAMS` that should be transcoded                             |
+| PHOTOPRISM_FFMPEG_MAP_VIDEO      | --ffmpeg-map-video      | 0:&zwnj;v:0                          | video `STREAMS` that should be transcoded                             |
+| PHOTOPRISM_FFMPEG_MAP_AUDIO      | --ffmpeg-map-audio      | 0:&zwnj;a:0                          | audio `STREAMS` that should be transcoded                             |
```

Comment:
The `?` at the end disappeared.

Reply:
It most certainly did. I've now replaced it